### DISCUSSION TITLE
Wallet Connect issue

### DISCUSSION THREAD

**Discussion by Aman-Antier** [WalletConnect/walletconnect-monorepo#3496] [Developer Support]
Hello dev, when i upload the build on my qa environment i am getting this error  (wagmi config found. Ensure you have set up a config:).but its working fine in my local environment. what is the error. please help me thanks you.

https://qa.ehx.exchange/

please check above the url 

![Screenshot from 2023-08-18 11-40-41](https://github.com/WalletConnect/walletconnect-monorepo/assets/109786461/b0611c1e-3de7-4dfd-9398-441b6bff141a)

![image](https://github.com/WalletConnect/walletconnect-monorepo/assets/109786461/082c4dc9-d60e-4937-928f-6a7fda87dfa9)

**Comment by glitch-txs** [comment]
Hi are you using Next.js? If so could you try setting swcMinify to false in the next.config.js file?

Also I'd suggest running
`npm i wagmi@latest viem@latest`

**Comment by Aman-Antier** [comment]
Hello Dev,
               I am using React js.So what should i do.Please suggest me.
Thank you.
